### PR TITLE
stargate: split client/ and cmd/

### PR DESCRIPTION
**Pull request by fedekunze**
- stargate: split proto and base files
- stargate: split app/ + codec/
- stargate: split types/
- stargate: split crypto/
- stargate: split x/evm/types/
- stargate: split x/evm/ (minus types/)
- stargate: split rpc/
- stargate: split client/ and cmd/

### CODE REVIEW

File: client/testnet.go [hunk 1]
```diff
@@ -252,37 +258,47 @@ func InitTestnet(
...
 			return err
 		}
 
-		txBytes, err := cdc.MarshalJSON(signedTx)
+		txBuilder := clientCtx.TxConfig.NewTxBuilder()
+		if err := txBuilder.SetMsgs(createValMsg); err != nil {
+			return err
+		}
+
+		txBuilder.SetMemo(memo)
```
Comment (noot) [inline client/testnet.go:270]: what does this do?

Comment (fedekunze) [reply client/testnet.go:270]: Just sets a memo or message to the transaction

File: cmd/ethermintd/genaccounts.go [hunk 2]
```diff
@@ -48,58 +44,66 @@ contain valid denominations. Accounts may optionally be supplied with vesting pa
...
 					inBuf,
-					hd.EthSecp256k1Options()...,
+					hd.EthSecp256k1Option(),
 				)
 				if err != nil {
 					return err
 				}
 
-				info, err := kb.Get(args[0])
+				// TODO: fix error, keyring.Info
```
Comment (noot) [inline cmd/ethermintd/genaccounts.go]: what is the error here?

**Review by noot — APPROVED** [review]
looks good

File: cmd/ethermintd/genaccounts.go (hunk 2 above)
Comment (fedekunze) [reply cmd/ethermintd/genaccounts.go]: I think this is a leftover comment, will remove

**Pull request closed without merging**